kafka-protocol-binding.md

```diff
@@ -76,16 +76,17 @@ the Kafka message's value section as-is, with the `content-type` header value
 declaring its media type; all other event attributes MUST be mapped to the
 Kafka message's [header section][kafka-message-header].
 
-Implementations that use Kafka 0.11.0.0 and above MAY use either _binary_
-or _structured_ modes. Implementations that use Kafka 0.10.x.x and below
-MUST use only use _structured_ mode and encode the event in JSON. This is
-because older versions of Kafka lacked support for message level headers.
+Implementations that use Kafka 0.11.0.0 and above MAY use either _binary_ or
+_structured_ modes. Implementations that use Kafka 0.10.x.x and below MUST only
+use _structured_ mode and encode the event in JSON. This is because older
+versions of Kafka lacked support for message level headers.
```

Comment:
just checking... this was just a formatting change, right?

Reply:
There was a typo. the word "use" occurred twice. I thought I could just fix it, while doing my edits.

Reply:
ah missed that - thanks!